app/src/main/java/io/curity/haapidemo/Configuration.kt

```diff
@@ -55,8 +55,7 @@ data class Configuration(
     val userInfoURI = baseURLString + userInfoEndpointPath
 
     init {
-        // Set these to `true` if you want the HAAPI SDK to log raw JSON responses.
```

Comment:
Should this be reverted?

Reply:
I will leave normal level logging on, which I think is a good default